worker.py

```diff
@@ -141,6 +143,10 @@ def run(self):
                 logger.warning('Malformed response received!')
                 self.error_code = 'RESTART'
                 self.restart()
+            except BannedAccount:
+                self.error_code = 'BANNED'
+                self.running = False
+                time.sleep(999999999999)
```

Comment:
It also was weird for me, cause on disabling workers `self.running = False` was enough, but here I added sleep, because my banned workers were restarting over and over, and still getting the same response.